h2o-algos/src/main/java/hex/ensemble/StackedEnsemble.java

```diff
@@ -214,51 +218,56 @@ public void computeImpl() {
     init(true);
     if (error_count() > 0)
         throw H2OModelBuilderIllegalArgumentException.makeFromBuilder(StackedEnsemble.this);
+      try {
```

Comment:
No changes in this block, only the try-catch block has been introduced, thus the code inside is shifted to the right.